pool/app/pool_user/entity.ml

```diff
@@ -37,19 +37,15 @@ module Status = struct
   ;;
 end
 
-let remove_whitespaces =
-  let open Re in
-  replace_string (space |> compile) ~by:""
-;;
-
 module EmailAddress = struct
   include Pool_model.Base.String
 
   let validate_characters email =
     let open Mrmime in
     match Mailbox.of_string email with
     | Ok _ -> Ok email
-    | Error _ -> Error Pool_message.(Error.Invalid Field.EmailAddress)
+    | Error _ ->
```

Comment:
LGTM. But I suggest to override the errors in the repo instead of the entity, if we are adding more information, to hide it from the users. 

Repo: 
```
module EmailAddress = struct
  include Entity.EmailAddress

  let t =
    make_caqti_type
      Caqti_type.string
      (fun email ->
        email
        |> create
        |> CCResult.map_err
             (CCFun.const
                Pool_message.(Error.InvalidWithInfo (Field.EmailAddress, email))))
      value
  ;;
end
```